### DISCUSSION TITLE
Where do we post issues with missing cards?

### DISCUSSION THREAD

**Discussion by dbouvier** [gaddlord/mtg.studio#467] [Q&A] [answered]
Where do we post issues with missing cards in the core DB?  The Issues tab here in GitHub appears to be code related issues.

This was prompted by the card, Serra Redeemer (normal art), missing from the Dominaria United (DMU) card set in the core DB.  The extended art version exists, but not the standard art version.

**Comment by gaddlord** [comment, marked as answer]
Whenever you create a new issue https://github.com/gaddlord/mtg.studio/issues/new/choose there is the option to select "Missing Card":

![image](https://user-images.githubusercontent.com/279005/210179831-5a5d33fb-8fc6-42e0-8ea0-a6152316b18c.png)
